soroban-env-host/src/host_object.rs

```diff
@@ -58,7 +64,7 @@ declare_host_object_type!(HostVec, Vec, Vec);
 declare_host_object_type!(u64, U64, U64);
 declare_host_object_type!(i64, I64, I64);
 declare_host_object_type!(Vec<u8>, Bytes, Bin);
-declare_host_object_type!(BigInt, BigInt, BigInt);
+declare_host_object_type!(MeteredBigInt, BigInt, BigInt);
 declare_host_object_type!(xdr::ScHash, Hash, Hash);
```

Comment:
See, here, I've replaced them with xdr::ScHash and xdr::PublicKey

Reply:
Got it!